docker-compose.yml

```diff
@@ -4,9 +4,10 @@ services:
   fedimintd_1:
     image: fedimint/fedimintd:master
     environment:
+      - FM_PASSWORD=pass
```

Comment:
Fedimintd containers should not require password. Rather, the password should be provided by the guardians via Setup UI.
```
// without FM_PASSWORD
fedimintd exits with code 1. says FM_PASSWORD env var required
```

Need to fix this, for the setup UI to work

```
// setup failure at DKG
Unable to write to data dir Os { code: 17, kind: AlreadyExists, message: "File exists" }
```

Reply:
fixed by https://github.com/fedimint/fedimint/pull/2587/files. I just had to clear my docker images and redownload to get the latest one which includes fix

Reply:
We could probably update the command to `Docker-compose pull && Docker-compose up`. Then it would always grab the latest. Unless we peg to a commit hash.